### ISSUE TITLE
hcloud_floating_ip: passed id from loop item is not of type int

### ISSUE THREAD

**Issue by sebb0** [thetechnick/hcloud-ansible#7]
I have exactly the some construct as the second example for this module in a playbook:
```yaml
- hcloud_floating_ip:
    state: list
  register: hcloud_floating_ips

- hcloud_floating_ip:
    id: "{{ item.id }}"
    server: "{{ inventory_hostname }}"
  loop: "{{ hcloud_floating_ips.floating_ips }}"
```
If I let this playbook run, I get:
```
failed: [node1] (item={'id': <removed>, 'description': 'Cloud Ingress', 'ip': '<removed>', 'type': 'ipv4',
'server_id': 0, 'home_location': 'nbg1'}) => {"changed": false, "item": {"description": "description",
"home_location": "nbg1", "id": <removed>, "ip": "<removed>", "server_id": 0, "type": "ipv4"},
"msg": "Cannot parse arguments file: json: cannot unmarshal string into Go struct field arguments.id of type int"}
```
Even if I add the int filter to the id of the floating ip like `id: "{{ item.id | int }}"`, I get the same result.
If I use the old style `with_items`, I get the same failure
If I use the id as a constant, everything works as expexted.

I guess this has something to do with the handling of variables in loops?

Another strange thing is by the way, that I always get back 0 as a server_id. Is this intended?

**Comment by thetechnick** [comment]
The first issue is caused by to strict parsing of the input values. The second because 0 is the default value of an int in Go.

I will upload a fix as soon as possible.

**Comment by thetechnick** [comment]
Fixed in #9, and just released in `v0.3.0`
